tests/test.fish

```diff
@@ -1,4 +1,4 @@
-#!/usr/local/bin/fish
+#!../test_root/bin/fish
```

Comment:
Yeah, I hadn't noticed that the makefile does `cd tests; ../test_root/bin/fish test.fish`. One or the other should be changed. Since I don't think we want to support running these tests outside of the makefile framework I'll replace these shebang lines with a comment to that effect.